test/membrane/life-red-proxies.spec.js

```diff
@@ -8,77 +8,383 @@ class ExoticObject {
     }
 }
 
+const array = [];
+const buffer = new ArrayBuffer(8);
+const dataView = new DataView(buffer);
+const float32Array = new Float32Array(buffer);
+const float64Array = new Float64Array(buffer);
+const int8Array = new Int8Array(buffer);
+const int16Array = new Int16Array(buffer);
+const int32Array = new Int32Array(buffer);
+const uint8Array = new Uint8Array(buffer);
+const regexp = /a/;
+const uint8ClampedArray = new Uint8ClampedArray(buffer);
+const uint16Array = new Uint16Array(buffer);
+const uint32Array = new Uint32Array(buffer);
+
 const plainObject = {
     x: 'uno',
 };
 
+const planObjectNullProto = {
```

Comment:
```suggestion
const plainObjectNullProto = {
```

Reply:
That naming did not go according to plain.